src/commands/ghCommands/status.js

```diff
@@ -0,0 +1,68 @@
+const GitHubApi = require('github');
+const chalk = require('chalk');
+const github = GitHubApi();
+
+module.exports = {
+	command: 'status',
+	description: 'Set a PR status',
+	builder: yarg =>
+		yarg
+			.options({
+				value: {
+					alias: 's', // s for 'status'. -v is conventionally a 'version' alias
+					demandOption: true,
+					type: 'string',
+					describe: 'The status value',
+					choices: ['pending', 'failure', 'error', 'success'],
+				},
+				commit: {
+					alias: 'c',
+					demandOption: true,
+					type: 'string',
+				},
+				token: {
```

Comment:
Question... Does this mean this will get spit out into travis logs when called for a repo? Really, just wondering if this should only be an env variable.

Reply:
I'll need to check, but I'm pretty sure that Travis will replace any 'secret' env var with `[secure]` in the logs

Reply:
🎉